redskyctl/internal/commands/experiments/arguments.go

```diff
@@ -42,7 +41,7 @@ func Completion(ctx context.Context, api experimentsv1alpha1.API, args []string,
 		return s, cobra.ShellCompDirectiveNoFileComp
 	}
 
-	// Assume we can parse now
+	// Assume we can names parse now
```

Comment:
should this be assume we can parse names now?

Reply:
Names parse, we must.